### DISCUSSION TITLE
Issues faced in the Ed Bootcamp March 2023

### DISCUSSION THREAD

**Discussion by gandham-santhosh** [Sunbird-Ed/Community#436] [Installation]
This discussion thread is to identify the issues faced by the participants while doing hands on in the bootcamp.
Request you to share the issues you have faced here in this thread which will help us to address them. Few of them are 

- Few participants were trying to set up Jenkins on their local laptop but failed as the host OS is Ubuntu 22.10 or Ubuntu 22.04
- As part of Jenkins setup, while creating the admin user for Jenkins it took time to save the settings in the Jenkins server. One has to reload the page multiple time to pass this step. 
- Hotel network started blocking requests to clone the github repositories so Jenkins jobs started failing. It took us some time to figure out that issue is not with the scripts or some permissions :) . Our hypothesis is that hotel network has firewall rules that blocks if there are frequent requests from public IP . Users has to switch to mobile hotspots to trigger Jenkins builds
- Artifact upload jobs were failing as docker hub details were not updated in the ansible variables. This was intentional so that participants understand the reason for failure :) . Many of the participants were able to understand the cause of failure

@AbhilashKD @abhishekb24 @adsingh7777 @agrawalanjali @Ashwiniev95 @asifbasha61 @BharathTarento @Debayanpaul1593 @Abhishekjain1106 @ankitpws @dhamoh @divyagovindaiah @Govind3877 @komalm @Priya-1123 @StartupSanatanaVenkatKL @Jyotiannoor @madhujaswal @manish050994 @manthanpandey23 @patilswati98 @pkranga @Premjith275 @ranvsin2 @Sanky-21 @Shakthieshwari @shiva6933 @snehal0904 @snehav05 @srajasimman @sudarshanswati @vatsaraj @vky2

**Comment by StartupSanatanaVenkatKL** [comment]
SURE
